collection+xattrs.go

```diff
@@ -90,17 +93,17 @@ func (c *Collection) DeleteWithMeta(_ context.Context, key string, oldCas CAS, n
 }
 
 // storeDocument performs a write to the underlying sqlite database of a document from a given event.
-func (c *Collection) storeDocument(txn *sql.Tx, e *event) error {
+func (c *Collection) storeDocument(txn *sql.Tx, e *event, revSeqNo uint64) error {
```

Comment:
Is there a reason not to add revSeqNo to event?  That seems like the struct used to manage document properties during read/write?

Reply:
event is typically dcp event, but I guess actually revSeqNo is there too, let me add it